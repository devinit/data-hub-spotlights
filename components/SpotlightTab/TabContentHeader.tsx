import React, { FunctionComponent } from 'react';

const TabContentHeader: FunctionComponent = ({ children }) => {
  return (
    <div className="tabs__content__header">
      { children }
    </div>
  );
};

export { TabContentHeader };
