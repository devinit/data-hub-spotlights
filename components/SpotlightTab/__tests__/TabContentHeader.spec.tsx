/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { TabContentHeader } from '../TabContentHeader';

describe('TabContentHeader', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<TabContentHeader />).toJSON();

    expect(renderer).toMatchSnapshot();
  });

  test('renders with children', () => {
    const renderer = TestRenderer.create(
      <TabContentHeader>
        <div>My Child</div>
      </TabContentHeader>
    ).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
