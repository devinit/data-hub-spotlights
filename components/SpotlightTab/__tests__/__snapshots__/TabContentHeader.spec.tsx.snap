// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContentHeader renders correctly 1`] = `
<div
  className="tabs__content__header"
/>
`;

exports[`TabContentHeader renders with children 1`] = `
<div
  className="tabs__content__header"
>
  <div>
    My Child
  </div>
</div>
`;
